【依頼】用語集ページのコードレビュー

toyo-otsukimana opened the pull request:
# What
- 用語集ページのレイアウトおよびスタイリングの実装をしました。

# Why
- ユーザーが手順ページの内容で分からない用語があった際にその用語の意味を閲覧できるようにするため。

## 実装したファイルと内容
- glossary.html：用語集ページのレイアウトを定義
- style.css：用語集ページに適用するスタイルを追記
- 用語集ページのスクリーンショット

![用語集ページのフルスクリーンショット_目次無し](https://github.com/toyo-otsukimana/projects/assets/172255345/a47109d3-031f-4772-a377-4951fdcaf38a)

- 修正後の用語集ページのスクリーンショット

![用語集ページ_20240619_プルリクエスト#8修_1](https://github.com/toyo-otsukimana/projects/assets/172255345/918fd136-53db-42ff-9f89-f3d2feb3e08a)

## テスト
- Chromeで正しく表示できていることを確認しました。
- accordionを使用したメニューを開閉する機能が正しく動作していることを確認しました。
- ページトップに戻るための機能が正しく動作していることを確認しました。

## 確認依頼
- 用語ページのレイアウトやスタイルに関するフィードバックをお願いします。
- 全体的なコードの品質や可読性についてレビューをお願いします。
- 特に問題がなければLGTMをプルリクエストのコメントにてお知らせください。

toyo-otsukimana committed:
Create glossary.html

用語集ページのベースとなるレイアウトを作成

toyo-otsukimana committed:
Creating a table of contents

手順ページの内容に合わせて対応する見出しまでジャンプできる目次

toyo-otsukimana committed:
Glossary page title and overview

用語集ページのタイトルと概要説明を追記

toyo-otsukimana committed:
Create a glossary page layout

用語集ページのレイアウトとサイトに記載する内容を記述

toyo-otsukimana committed:
Styling the glossary page

用語集ページのスタイリング

toyo-otsukimana committed:
Adding borders and adding class names to h3 tags

罫線の追加とh3タグのクラス名を追記

toyo-otsukimana committed:
Set links to navigate to the menu for each term

目次から各用語に遷移できるようにリンクを設定

toyo-otsukimana committed:
Implement a button to return to the top of the page

ページトップに戻るためのボタンを実装

toyo-otsukimana committed:
Remove unnecessary JavaScript calls

JavaScriptの処理を呼び出す機能のうち、不要なcode.jsの呼び出しを削除

toyo-otsukimana committed:
Comment out the table of contents

HTMLリファレンスページなどとレイアウトを合わせるために目次をコメントアウト

Tonos4ki commented:
お疲れ様です。
お待たせして申し訳ありません。
ページ作成ありがとうございます。

レイアウトの修正をお願いします。

- 大見出し、文章にwidth75%とmargin: 0 autoを追記
- 文章にpadding 10px 0 15px 20pxを追記
※他のページと見比べた時に見出しや文章が位置がずれているため。

- item__titleの見出しのフォントの大きさを23pxに変更
※少々大きすぎると感じたため。

toyo-otsukimana commented:
お疲れ様です。
レイアウトの修正について、どの部分を修正すればよいか少しわかりにくいと感じました。恐れ入りますが、具体的なファイル名や行番号、クラス名などを示していただけると助かります。

また、修正箇所のコメントについて下記の手順でコメントしていただけると、修正箇所が明確かつ修正作業がスムーズに進められると考えております。
お手数ではございますが、よろしくお願いいたします。
```
1.リポジトリ内にある「Pull Rrequest」をクリックする
2.プルリクエストのリストで、行コメントをしたいプルリクエストをクリックする
3.「Files changed」をクリックする
4.コメントを追加するコード行にカーソルを合わせて青い＋のアイコンをクリックする
5.コメントフィールドにコメントを記述する
6.コメントの記述が完了したら、「Start a review」をクリックする
7.複数個所にコメントしたい場合は、4～6の操作を繰り返す
8.すべてのコメントが完了したら「review」をクリックする
9.コメントフィールドに必要に応じてコメントを記述する（この部分は何も入力していない状態でも送信できる）
10.記述が完了したら、「submit review」をクリックする
※3.で「Commit」をクリックしてもほぼ同じ操作でコメントできます。
```

Tonos4ki commented on style.css:
```diff
@@ -571,5 +572,49 @@ strong {
+}
+
+.item__title{
+  font-size: 25px;
```
598行目
font-size: 23px;

Tonos4ki commented:
お疲れ様です。

先ほどのご案内が分かりにくく、申し訳ございませんでした。ご指摘いただきありがとうございます。上記に修正箇所を明確にしました。また、修正箇所にはコメントを追加しましたので、ご確認の上修正を進めていただければ幸いです。

toyo-otsukimana committed:
Fixed the layout of the main heading and text, and added styling.

修正箇所のスタイリングの追記(glossary-title , glossary-about)とそれに伴うレイアウトの修正

toyo-otsukimana committed:
Change the title of the navigation header

ナビゲーションヘッダーを最新のブランチに共有されている内容に合わせました。

toyo-otsukimana commented:
お疲れ様です。
昨日にご提案された修正内容につきまして、先ほど用語集ページのソースコードに反映を完了させました。

該当コミット：
1. Fixed the layout of the main heading and text, and added styling.（修正内容の反映）
2. Change the title of the navigation header（最新のmainブランチにおけるナビゲーションヘッダーの内容に修正）

また、コードレビューに関しまして、ご確認いただきたい点がございます。現在、修正箇所に対するコメントがこちら側で確認できない状態になっております。お手数をおかけしますが、保存されたコメントが保留（Pending）状態になっていないか、ご確認いただけますでしょうか。

ご不明点や追加の情報が必要な場合は、お知らせください。何卒よろしくお願い申し上げます。

Tonos4ki commented on glossary.html:
```diff
@@ -0,0 +1,344 @@
+      この用語集は、Webサイトの作成プロセスにおける基本的な用語とその説明をまとめたものです。<br>
+      初心者の方にも分かりやすいように「手順」ページに基づき、「企画・計画」・「デザイン」・「開発」「テストと公開」・「メンテナンス」の各段階に関連する用語を網羅しています。<br>
+      Webサイト作成の際に役立つ情報を提供しておりますので、ご活用ください。
+    </div>
```
1. 26行目

- width: 75%;
- margin: 0 auto;

2. 28-30行目

-   width: 75%;
-   padding: 10px 0 15px 20px;
-   margin: 0 auto;

Tonos4ki commented:
お疲れ様です。
送信されていなかったようです。
お手数ですがもう一度確認の程よろしくお願いします。

toyo-otsukimana commented:
お疲れ様です。
昨日にご提案された修正内容につきまして、先ほど用語集ページのソースコードに反映を完了させました。

該当コミット：

Fixed the layout of the main heading and text, and added styling.（修正内容の反映）
Change the title of the navigation header（最新のmainブランチにおけるナビゲーションヘッダーの内容に修正）

お手数ではございますが、ご査収のほどよろしくお願いいたします。

toyo-otsukimana committed:
Copyright correction

Tonos4ki commented:
お疲れさまです。
修正ありがとうございます。
修正後のスクリーンショット一番上に追加されてたんですね。
気づかなくて申し訳ありませんでした。

LGTMとさせていただきます。

toyo-otsukimana committed:
Merge branch 'main' into Creating-a-Glossary-Page
